qemu/tests/cfg/guest_iommu_group.cfg

```diff
@@ -0,0 +1,41 @@
+# Please add "iommu=pt intel_iommu=on" to host kernel line
+- guest_iommu_group: 
+    virt_test_type = qemu
+    type = guest_iommu_group
+    only virtio_blk virtio_net virtio_scsi
```

Comment:
I think this is not a requirement in this case, depends on whether users want to use which one.